ceph.spec.in

```diff
@@ -1411,7 +1453,34 @@ fi
 
 %files mgr
 %{_bindir}/ceph-mgr
-%{_libdir}/ceph/mgr
+%{_libdir}/ceph/mgr/ansible
+%{_libdir}/ceph/mgr/balancer
+%{_libdir}/ceph/mgr/crash
+%{_libdir}/ceph/mgr/dashboard
+%{_libdir}/ceph/mgr/deepsea
+%{_libdir}/ceph/mgr/devicehealth
+%{_libdir}/ceph/mgr/influx
+%{_libdir}/ceph/mgr/insights
+%{_libdir}/ceph/mgr/iostat
+%{_libdir}/ceph/mgr/localpool
+%{_libdir}/ceph/mgr/mgr_module.*
+%{_libdir}/ceph/mgr/mgr_util.*
+%{_libdir}/ceph/mgr/orchestrator_cli
+%{_libdir}/ceph/mgr/orchestrator.*
```

Comment:
`%{_libdir}/ceph/mgr/*.py` ?

Reply:
We could, but I don't see that it's necessarily better than <individual file>.*